components/script/dom/document.rs

```diff
@@ -4861,7 +4859,7 @@ impl DocumentMethods for Document {
 
     #[allow(unsafe_code)]
     // https://html.spec.whatwg.org/multipage/#dom-tree-accessors:dom-document-nameditem-filter
```

Comment:
Do you mind updating the comment here to be valid rustdoc while you are here?

```
/// <https://html.spec.whatwg.org/multipage/#dom-tree-accessors:dom-document-nameditem-filter>
```

Reply:
Absolutely!